src/System.Drawing.Common/src/System.Drawing.Common.csproj

```diff
@@ -166,9 +166,6 @@
     <Compile Condition="'$(TargetGroup)' == 'netcoreapp2.0'" Include="$(CommonPath)\System\Drawing\ColorUtil.netcoreapp20.cs">
```

Comment:
Is this still needed after this change?

Reply:
Not once #39275 goes in. There is some overlap here.